src/lib/marshal/workflow/reader.ts

```diff
@@ -111,68 +103,6 @@ const validateExtractedFilePath = (
   return undefined;
 };
 
-/*
- * Read the file at the given path if it exists, validate the content as
- * applicable, and return the content string or an error.
- */
-type ExtractedFileContent = string | ParsedJson;
```

Comment:
Abstracted this in `shared/helpers`, as we do now read extracted files for layouts as well.